app/core/config.py

```diff
@@ -41,10 +41,10 @@
 
 
 # logging configuration
-
+logging.basicConfig(level=logging.DEBUG)
 LOGGING_LEVEL = logging.DEBUG if DEBUG else logging.INFO
 LOGGERS = ("uvicorn.asgi", "uvicorn.access")
-
+log = logging.getLogger("gunicorn.access")
```

Comment:
Is it `uvicorn.access` or `gunicorn.access` ?

Reply:
I removed it but it's in most of our projects just in case we start using gunicorn but it doesn't have an impact otherwise.